qutebrowser.json

```diff
@@ -4,11 +4,11 @@
     "version": "1.1.1",
 	"architecture": {
         "64bit": {
-            "url": "https://github.com/qutebrowser/qutebrowser/releases/download/v1.1.1/qutebrowser-1.1.1-windows-standalone-amd64.zip",
+            "url": "https://github.com/qutebrowser/qutebrowser/releases/download/v1.1.1/qutebrowser-1.1.1-windows-standalone-amd64.zip#/qutebrowser.exe",
```

Comment:
adding `#/qutebrowser.exe` instructs scoop to rename the file from `qutebrowser-1.1.1-windows-standalone-amd64.zip` to `qutebrowser.exe`. This won't work because it doesn't extract exe files :)

Reply:
First time I'm posting something without checking and boom! I'm coming to close pull request and I'm seeing your commit. 😆 🤦‍♂️ Closing pull request in order to fix errors!

Reply:
Just change it, commit and push to the same branch. it will show up here 😄

Reply:
I'm just punishing myself for my mistakes! 🤣 I hope that this is fine now.

Reply:
Btw, can you look extract_dir part? x86 and x64 folder names are different in zip files.

```
qutebrowser-ver-win-standalone-arch.zip
└───qutebrowser-ver-arch
            └───files

```
Is my solution in "architecture" is correct or not? (Installed in my system (x64) with latest push is working)